base/applications/notepad/settings.c

```diff
@@ -102,12 +102,11 @@ static BOOL QueryString(HKEY hKey, LPCTSTR pszValueName, LPTSTR pszResult, DWORD
  *
  *  Load settings from registry HKCU\Software\Microsoft\Notepad.
  */
-void NOTEPAD_LoadSettingsFromRegistry(void)
+void NOTEPAD_LoadSettingsFromRegistry(WINDOWPLACEMENT*pWP)
 {
     HKEY hKey;
     HFONT hFont;
-    DWORD dwPointSize, cx, cy;
-    DWORD cxScreen = GetSystemMetrics(SM_CXSCREEN), cyScreen = GetSystemMetrics(SM_CYSCREEN);
+    DWORD dwPointSize, x = CW_USEDEFAULT, y = CW_USEDEFAULT, cx = 0, cy = 0;
```

Comment:
```suggestion
    DWORD dwPointSize;
    DWORD x = CW_USEDEFAULT, y = CW_USEDEFAULT;
    DWORD cx = 0, cy = 0;
```

Reply:
I'm refusing to put x and cx on separate lines, that is just overkill.